root/static/scripts/common/entity.js

```diff
@@ -273,15 +273,18 @@ import MB from './MB.js';
     toJSON() {
       return Object.assign(
         super.toJSON(),
-        {iso_3166_1_codes: this.iso_3166_1_codes || []},
+        {
+          containment: this.containment || [],
+          iso_3166_1_codes: this.iso_3166_1_codes || [],
```

Comment:
For safety we might want to add lines for the `*_2_*` and `*_3_*` properties too. Since, if those exist, they would be accessible from the instance but not the `toJSON` return value.